controls/gridview/exporting-data/spread-export.md

```diff
@@ -192,9 +209,13 @@ __ExportViewDefinition:__  Gets or sets a value indicating whether to export the
 
```

Comment:
The bullet list in the ## Properties section of the document is ginormous. Gigantic bullet lists have limited practical use. I doubt that anyone would read more than 8 bullets. If I see a section with 20 bullet items, I wouldn't even read 3 of them. Some suggestions:
- Can we group these bullets into parent paragraphs? Each parent paragraph can provide some general info about the related features.
- Instead of a bunch of bullets, can we have a simple scenario that demonstrates how these features can be used in practice?
- Another option would be a table.
- You could create a document that has all the features. In the Overview doc, you could list only the highlights.